py/torch_tensorrt/dynamo/conversion/converter_utils.py

```diff
@@ -511,3 +511,35 @@ def to_numpy(
         raise AssertionError(
             f"to_numpy can only be called on None, bool, int, float, np.ndarray, or torch.Tensor, got: {value}"
         )
+
+
+def flatten_dims(
```

Comment:
Add a test to `test_converter_utils.py`, which tests this function on a simple Numpy array or Torch Tensor, to validate the output shape is correct given the input shape.